pyproject.toml

```diff
@@ -13,6 +13,8 @@ fastapi = "^0.115.11"
 uvicorn = "^0.34.0"
 pydantic = "^2.10.6"
 pandas = "^2.1.4"
+google-cloud-logging = "^3.9.0"
+survey-assist-utils = {git = "https://github.com/ONSdigital/survey-assist-utils.git"}
```

Comment:
Can we tag the version to v.0.0.1 (I have spun a release of survey-assist-utils).